interfaces/hydra_endpoints/v0/README.md

```diff
@@ -0,0 +1,75 @@
+# `hydra_endpoints`
+
+## Usage
+
+This relation interface describes the expected behavior of provider ([Ory Hydra](https://github.com/canonical/hydra-operator))
+and requirer ([Ory Kratos](https://github.com/canonical/kratos-operator)) sides of relation.
```

Comment:
Here I'd use the same as other relation interfaces: `This relation interface describes the expected behavior of charms claiming to be able to provide or consume an hydra endpoint`